extensions/data-transfer/portability-data-transfer-koofr/src/main/java/org/datatransferproject/transfer/koofr/photos/KoofrPhotosImporter.java

```diff
@@ -54,7 +56,6 @@
 public class KoofrPhotosImporter
     implements Importer<TokensAndUrlAuthData, PhotosContainerResource> {
 
-  private static final String SKIPPED_FILE_RESULT_FORMAT = "skipped-%s";
```

Comment:
ouch, we still use this string afaik

Reply:
It wasn't being used though declared. Anyway, I added it back and replaced the usages.